client/src/docs-ui/search-bar/search-bar.tsx

```diff
@@ -0,0 +1,42 @@
+import {Component, Host, h, Build, Listen} from "@stencil/core";
+import {searchStyle} from "./search-bar.style";
+import {transformData} from "../../utils/transform-search-data";
+
+@Component({tag: "docs-search-bar", shadow: false})
+export class DocsSearchBar {
+  initDocSearch() {
+    if (Build.isBrowser) {
+      // @ts-ignore
+      docsearch({
+        apiKey: "24d37f059982b2f5ecf829afe93aed40",
```

Comment:
Is there a way we can adjust this to be a CONSTANT? Or is this going to change over time?

Reply:
Fair enough––will move into constants